scripts/extensions/markForUser/src/extension.tsx

```diff
@@ -48,8 +48,7 @@ const extension: IExtension = {
                     },
                     notifications: {
                         'item:marked': {
-                            type: 'desktop',
-                            label: gettext('open item'),
+                            name: gettext('Mark for User'),
```

Comment:
does this unified notification notify both of marking and unmarking? Check with if API exposes this information and coordinate with Petr if not

Reply:
Yes, works for marking and unmarking. Only difference between the two previously was the body message, handler was indeed the same one.